managed/src/main/java/com/yugabyte/yw/models/Universe.java

```diff
@@ -895,4 +895,14 @@ public void incrementVersion() {
   public static boolean existsCertificate(UUID certUUID, UUID customerUUID) {
     return universeDetailsIfCertsExists(certUUID, customerUUID).size() != 0;
   }
+
+  static boolean isUniversePaused(UUID uuid) {
+    try {
```

Comment:
logic is correct.. But you need to pull the latest. 
Use newly added  `maybeGet` method so you dont have to depend on exception for control flow.